node/tessel-export.js

```diff
@@ -578,7 +585,8 @@ Tessel.Pin.prototype.addListener = function(mode, callback) {
 Tessel.Pin.prototype.on = Tessel.Pin.prototype.addListener;
 
 Tessel.Pin.prototype._setInterruptMode = function(mode) {
-  this.interruptMode = mode;
+  // rise and fall events will be emitted by change event
+  (mode === 'rise' || mode === 'fall') ? this.interruptMode = 'change' : this.interruptMode = mode;
```

Comment:
Is it not common to use the ternary operator like this? I'm a relative newbie so I appreciate all of the feedback and help.